imports/plugins/core/accounts/server/methods/customizeLoginErrorMessage.js

```diff
@@ -0,0 +1,27 @@
+import Logger from "@reactioncommerce/logger";
+import { check } from "meteor/check";
+import getGraphQLContextInMeteorMethod from "/imports/plugins/core/graphql/server/getGraphQLContextInMeteorMethod";
+
+/**
+ * @name accounts/customizeLoginErrorMessage
+ * @memberof Accounts/Methods
+ * @method
+ * @param {String} username - username/email entered by a user
+ * @param {Object} error - error object returned by Meteor.loginWithPassword
+ * @returns {Object} custom error object
+ */
+export default function customizeLoginErrorMessage(username, error) {
+  check(username, String);
+  check(error, Object);
+
+  let customError = error;
+  const context = Promise.await(getGraphQLContextInMeteorMethod());
+  const { getFunctionsOfType } = context;
+  const customizeErrorFunctions = getFunctionsOfType("customizeLoginErrorMessage");
+  if (customizeErrorFunctions.length > 0) {
```

Comment:
Should we be calling all the functions in the array instead of just the first?

Reply:
Not sure to be honest as I there needs to be only one result anyway. I just got inspired by this:

https://github.com/reactioncommerce/reaction/blob/dec31982854ca4b1e907319ac138d49c3eac6dc8/imports/plugins/core/orders/server/no-meteor/mutations/placeOrder.js#L275-L292

where the `getFuctionsOfType` only applies the first function.

Reply:
It's fine for code to choose to (and document that it will) only call the first function. But in this case, I think @focusaurus is right that it would make more sense to call them all in series and stop as soon as one returns something. That way different plugins can customize different errors.